### DISCUSSION TITLE
JSON parse error

### DISCUSSION THREAD

**Discussion by alex473x** [rern/rAudio#229] [Fixed Issues]
Hello all and thanks for great job you do.
After using rAudio for few days with success, I'm now stuck at start with a JSON parse error message:
`Error: JSON.parse: unexpected character at line 1 column 98 of the JSON data{ "player" : "mpd" , "btclient" : false , "consume" : false , "control" : "Digital" , "counts" : ,▮ "lcd" : false , "librandom" : false , "playlistlength" : 1 , "playlists" : 0 , "relays" : false , "relayson" : false , "updateaddons" : true , "volume" : 100 , "volumemute" : 0 , "repeat" : false , "random" : true , "single" : false , "elapsed" : false , "file" : "http://stream.radioparadise.com/world-etc-flacm" , "song" : 0 , "state" : "stop" , "timestamp" : 1614497532132 , "updating_db" : false , "volumemute" : 0 , "Album" : "http://stream.radioparadise.com/world-etc-flacm" , "Artist" : "Radio Paradise - World Mix" , "Name" : "Radio Paradise World-Etc Mix (flac) - playlist: radioparadise.com" , "Time" : false , "Title" : "" , "webradio" : true , "sampling" : "1/1 • 16 bit 44.1 kHz 1.41 Mbit/s" , "coverartradio" : "/data/webradiosimg/http:||stream.radioparadise.com|world-etc-flacm.1614497532.jpg" , "coverart" : ""}`

Seems a wrong character is inserted between "counts" and "lcd". How can I solve this issue ?
I tried to reboot in ssh, look and search for all *.json files content without success.

**Comment by rern** [comment]
Try:
`rm /srv/http/data/mpd/*`
Then reboot.

**Reply by alex473x** [reply]
thanks !

**Reply by yopmismo** [reply]
Hello, I had the same problem, but I never could move from this JSON error, no matter what I do.
I boot without nothing connected, I tried using a display with mouse and keyboard, same error.
I connected the 250gb hard drive, nothing.
I tried connecting DAC only, nothing.
Combination of both, nothing.
The problem started when I add some tracks to my 12 TB RAID, then I remounted it and then the Rune never come back.
I asume I'm loosing my playlist if I start from zero again.
Other option to try different to erase and reflash?

**Reply by fillods** [reply]
At the JSON level, the error does not come from the weird character, but because of syntax error : `  "counts" : , `  i.e. the number is missing.

Have a look at beginning of script `srv/http/bash/status.sh` , it should be filled by the content of the file `/srv/http/data/mpd/counts` .

BTW, what version of rAudio are you running ?

**Reply by yopmismo** [reply]
I can't see it now, because I'm stolled in JSON error, but I downloaded it on March 01, 2021 and I never updated it.

**Reply by rollant** [reply]
Got this error today because `/srv/http/data/mpd/counts` was empty.
Looks like that's because `$( cat /srv/http/data/mpd/counts 2> /dev/null || echo false )` returns `false` only if `cat` exits with error
